flake.nix

```diff
@@ -3,14 +3,65 @@
 
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
-    nixpkgs.url = "nixpkgs/21.11";
+    nixpkgs.url = "nixpkgs/release-22.11";
```

Comment:
Updated nixpkgs here. Note the use of the "release-" prefix. The previously used tag "21.11" is not a branch and seems to be never updated in the `nixpkgs` repository.